mergify_engine/context.py

```diff
@@ -1601,7 +1603,13 @@ class PullRequest(BasePullRequest):
     }
 
     async def __getattr__(self, name: str) -> ContextAttributeType:
-        return await self.context._get_consolidated_data(name.replace("_", "-"))
+        data_name = name
+        if name == "body-raw":
+            data_name = "body"
+        data = await self.context._get_consolidated_data(data_name.replace("_", "-"))
+        if name == "body":
+            return re.sub("(<!--.*?-->)", "", typing.cast(str, data))
```

Comment:
I'd do it from `_get_consolidate_data`, no? sounds cleaner

Reply:
`_get_consolidate_data()` is also used with the `rules.filter` code. If we change the body there, rules trying to match something in a comment will fail. But maybe we don't care.

Reply:
I don't think we care indeed.
We might want to update the doc to include both `body` and `body-raw` though.